File: CS/CustomizeWinFormsTemplateExample.Module/Controllers/MyViewController.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;

namespace CustomizeWinFormsTemplateExample.Module.Controllers {
    public class MyViewController : ViewController {
        public MyViewController() {
            SimpleAction myAction = new SimpleAction(this, "MyAction", "My Actions");
            myAction.ImageName = "Action_SimpleAction";
        }
    }
}

File: VB/CustomizeWinFormsTemplateExample.Module/Controllers/MyViewController.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions

Namespace CustomizeWinFormsTemplateExample.Module.Controllers
    Public Class MyViewController
        Inherits ViewController

        Public Sub New()
            Dim myAction As New SimpleAction(Me, "MyAction", "My Actions")
            myAction.ImageName = "Action_SimpleAction"
        End Sub
    End Class
End Namespace
